### DISCUSSION TITLE
Is it intentional that the latest 1.0.9 release needs libQt5Pas from that comes from lazarus 2.3?

### DISCUSSION THREAD

**Discussion by corvus1** [doublecmd/doublecmd#701] [General]
It fails to link with libqt5pas-1.2.9 that comes with the release version lazarus-2.2. The way they do releases kinda saddens me.

**Comment by alexx2000** [comment]
It does not matter for DC itself. It depends on which Lazarus version do you use to build. If you use Lazarus 2.3 then you should use libqt5pas from Lazarus 2.3 and vice versa.

I build release version using Lazarus 2.2.2.

**Reply by corvus1** [reply]
You're right, that's exactly the mistake that I made. 
Why do you use 2.2.2 and not 2.2.4. which is the latest release, I think. Is there a particular reason?

**Comment by alexx2000** [comment]
Because I didn't update build server yet.

**Comment by corvus1** [comment]
Ah, okay. 
I just thought for a second maybe there's something good about 2.2.2 (or bad about >2.2.2), because conincidentally, 2.2.2 is also the latest in gentoo official repo. 

I'm still determined to package DC _properly_. I'm making my own little overlay.
